faststream/confluent/client.py

```diff
@@ -102,9 +114,21 @@ def __init__(
 
         self.producer = Producer(self.config, logger=self.logger)
 
+        self.__running = True
+        self._poll_thread = Thread(target=self._poll_loop)
```

Comment:
Wa copied from this example: https://github.com/confluentinc/confluent-kafka-python/blob/master/examples/asyncio_example.py